src/main/groovy/org/transmartproject/core/dataquery/highdim/tworegion/EventValues.groovy

```diff
@@ -0,0 +1,16 @@
+package org.transmartproject.core.dataquery.highdim.tworegion
+
+/**
+ * Represents metadata about one event, possibly linking to multiple junctions
+ */
+interface EventValues {
+    /**
+     * deletion, inversion, duplication,... Type from http://cgatools.sourceforge.net/docs/1.8.0/cgatools-command-line-reference.html#junctions2events
+     */
+    String getCgaType()
+    /**
+     * inter/intra chromosomal inversion/translocation: http://sourceforge.net/p/soapfuse/wiki/classification-of-fusions.for.SOAPfuse/
```

Comment:
This is not the proper way to include links in JavaDoc. See http://stackoverflow.com/a/10683345/127724